### PR TITLE
Patch 2

### PR DESCRIPTION
**Pull request by cmakinson**
Cody Makinson

1. I want to analyze NHL data from the birth of the NHL original 6 hockey teams.
2. I really enjoyed pokemon growing up, so I would use the data we previously used in an assignment. I want to see if there is a type bias, among other statitical analysis.
 3. Weather data. It is cyclical and a different kind of data than the other two.
https://www.justsportsstats.com/hockeyindex.php?league= *NHL I just need to download in excel* https://pokemondb.net/pokedex/all * I would have to play around a little to make into .csv file* PRECIP_HLY_sample_csv

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,3 +49,12 @@ Here are some web pages that you may need to refer to:
       -https://www.kaggle.com/tathor/yugioh-trading-cards-dataset/data
       - YuGiOh is a card game that I have a lot of interest in and I think it would be fun to explore the most common card type or              monster attributes given the over 6,000 cards included in this data set.
   
+
+Cody Makinson
```
Comment (cowang) [inline README.md]: Good. You have a bullet point for each dataset. Now organize each URL as a bullet point indented underneath one of those. The idea is that someone should be able to look at the information for each dataset separately.

File: README.md [hunk 2]
```diff
@@ -49,3 +49,12 @@ Here are some web pages that you may need to refer to:
...
+1. I want to analyze NHL data from the birth of the NHL original 6 hockey teams.
+2. I really enjoyed pokemon growing up, so I would use the data we previously used in an assignment. I want to see if there is a type bias, among other statitical analysis.
+3.Weather data. It is cyclical and a different kind of data than the other two.
+
+https://www.justsportsstats.com/hockeyindex.php?league=NHL *I just need to download in excel*
```
Comment (cowang) [inline README.md]: I wasn't able to see how to download any of these as files. You should prove to yourself that they are available and take a look at the files you are thinking about using in Excel to get a feel for the data.

File: README.md (hunk 1 above)
Comment (cowang) [reply README.md]: You should also put a dash in front of "Cody" and line it up under "- George Cowan" to make your name be at the right level of the outline.

**Comment by cmakinson** [comment]
Ok I will fix these little mistakes. Other than that did I upload stuff
correctly? Sorry I never used GitHub before and this whole online thing for
all my classes is slowly getting to me lol. So I apologize if I ever sound
stressed because... well the honest truth I am stressed. Thank you for
being patient and understanding and handling the switch to online so well.
I may need another phone conversation this coming up week for advice with
projects and GitHub.
All the best,
Cody

**Comment by cowang** [comment]
Your upload worked fine.
Let me check my calendar. Ah, you're lucky -- I have absolutely nothing on my calendar for the next week except sleeping late. Imagine that! Why don't you suggest a time that would be good for you for a phone meeting, after 10AM.
And let's move this conversation to Canvas.

File: _config.yml [hunk 3]
```diff
@@ -0,0 +1 @@
+theme: jekyll-theme-architect
```
Comment (cowang) [inline _config.yml]: It looks like you were experimenting with the design of the web page. Did you think the Architect theme looked better?

**Pull request merged**